spec/views/users/show_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+RSpec.describe 'User', type: :feature do
+  before(:each) do
+    @user = User.create(
+      name: 'Kiploman',
+      bio: 'Graphic designer',
+      photo: 'http://hello.com/org.png',
+      posts_counter: 0
+    )
+  end
+  describe 'show page' do
+    it "has users's username." do
+      visit "/users/#{@user.id}"
+      expect(page).to have_content(@user.name)
+    end
+    it "has users's bio." do
+      visit "/users/#{@user.id}"
+      expect(page).to have_content(@user.bio)
+    end
```

Comment:
- [ ] Please write the tests for the 3 scenarios shown below because they are milestone requirements👇 
* Tests that asserts you can see a button that lets me view all of a user's posts.
* Tests that redirects to the post's show page when you click on a user's post.
* Tests that redirects to the user's post's index page when you click on a **see all posts**
![image](https://user-images.githubusercontent.com/46839548/223722264-da571af6-c118-4691-bcfb-e242b25b1ab3.png)